src/python/WMCore/DataStructs/DASDocument.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, service):
         """
         Make an empty dictionary representing a DAS document
         """
-        dict = {"request_timestamp": 0,
```

Comment:
@amaltaro This change was not suggested by futurize, it was mine.
Is the redefinition of the reserved keyword `dict` intentional? It may not cause any problem, but safe is better than sorry.
I also took the liberty of renaming it to `dasdoc`, please suggest a new and better name if you do not like this one.

Reply:
This module is not used anywhere (and apparently not by any of the external services either). We can keep these changes, and I will delete this module in another PR. Or we delete it in another PR, and you remove these changes.
I assume the first option gives you less work.